src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -261,43 +277,6 @@ export default class MutationMapperStore extends DefaultMutationMapperStore {
         return lazyMobXTableSort(this.mergedAlignmentData, sortMetric, false);
     }
 
-    @computed get transcriptsByTranscriptId(): {
-        [transcriptId: string]: EnsemblTranscript;
-    } {
-        return _.keyBy(
-            this.allTranscripts.result as EnsemblTranscript[],
-            transcript => transcript.transcriptId
-        );
-    }
-
-    @computed get mutationsByTranscriptId(): {
```

Comment:
no need to override this one anymore, we override `getAnnotatedMutations` instead.